### PR TITLE
interrupt build scripts for docker files if failure occurs

### PR DESCRIPTION
**Pull request by riuvshin**
### What does this PR do?
interupt build scripts for docker files if failure occurs

#### Changelog
interupt build scripts for docker files if failure occurs

### CODE REVIEW

File: dockerfiles/cli/build.sh [hunk 1]
```diff
@@ -8,6 +8,8 @@
 # Contributors:
 #   Florent Benoit - Initial Implementation
 
+set -e
```
Comment (benoitf) [inline dockerfiles/cli/build.sh]: do we need it as we include build.include ?

Comment (riuvshin) [reply dockerfiles/cli/build.sh]: not sure, but just in case I added this :)

Comment (benoitf) [reply dockerfiles/cli/build.sh]: I think we can drop on all .sh that imports build.include

also you should add `set -u` as well in build.include

Comment (riuvshin) [reply dockerfiles/cli/build.sh]: are you sure to remove on sh? my issue was that this https://github.com/codenvy/codenvy/blob/master/dockerfiles/agents/build.sh#L16 line failed but script passed. it was before build.include was used.

you mean `. ${DIR}/../build.include`  will apply `set -e` for build.sh ?

Comment (riuvshin) [reply dockerfiles/cli/build.sh]: tested, `. ${DIR}/../build.include` will apply `set -e`

Comment (benoitf) [reply dockerfiles/cli/build.sh]: yes

Comment (riuvshin) [reply dockerfiles/cli/build.sh]: done

**Pull request merged**